calico_containers/calico_ctl/node.py

```diff
@@ -319,32 +341,91 @@ def node_start(node_image, log_dir, ip, ip6, as_num, detach, kubernetes, rkt,
         environment=environment,
         host_config=host_config,
         volumes=["/proc_host",
```

Comment:
Do we still need proc_host? I think it might have been a powerstrip thing. Now might be a good time to remove it...